packages/account/src/utils/Types.ts

```diff
@@ -39,6 +39,13 @@ export interface BalancePayload {
   formattedAmount: string;
 }
 
+export interface WithdrawalRequest {
+  /** The address of the tokenAddress */
+  address: Hex;
```

Comment:
could name destination or recipient or receiver

Reply:
No this is the asset address, not the recipient.